src/constants/settings.ts

```diff
@@ -0,0 +1,84 @@
+import {ComponentType, SVGProps} from "react";
+import {GeneralSettings, Participants, SettingsAppearance, SettingsFeedback, Share, FileDefault} from "components/Icon";
+import {Color} from "./colors";
+
+export const MOBILE_BREAKPOINT = 920;
```

Comment:
that magic value has been in the code since the very first version of the setting component added in 2022 (1f88c5e8e77573f702fa95b0cc2b326082bf5734), I just decided to outsource it and clarify its use in the code.